app/frontend/src/app/@theme/theme.module.ts

```diff
@@ -29,21 +32,24 @@ const BASE_COMPONENTS = [
   LayoutButtonComponent,
 ];
 
+const MODULES = [
```

Comment:
Not all modules should be exported
* CommonModule
* FontAwesomeModule
* ReactiveFormsModule